index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Newsweek Replica</title>
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+	<link href="styles.css" rel="stylesheet">
+</head>
+<body>
+	<header>
+		<div class="container-fluid d-flex justify-content-between redtopbar">
+				<div class="col-lg-2 d-none d-lg-inline align-self-end">
+					<h4 class="nav-text date">Wed, Jan 20, 2021</h4>
+				</div>
+				<div class="col-lg-5 col-sm-3">
+					<a class="logo-link" href="#"><img class="logo" alt="Newsweek Logo" src="https://upload.wikimedia.org/wikipedia/commons/d/db/Newsweek_Logo.svg"></a>
+				</div>
+				<div class="col-5 d-flex justify-content-lg-center justify-content-around">
+					<div class="d-flex align-self-center">
+						<button class="nav-text login d-none d-md-inline" type="button">LOGIN</button>
+						<button class="login-icon d-md-none d-inline" type="button"><img class="login-icon" src="https://g.newsweek.com/www/images/user-w.svg" alt="login icon"></button>
+					</div>
+					<div class="d-flex align-self-center flex-nowrap">
+						<button class="nav-text subscribe" type="button"><span class="subButton bg-yellow">SUBSCRIBE &gt;</span></button>
+					</div>
```

Comment:
:heavy_exclamation_mark: The subscribe button should have a link

Reply:
I get a linter error when I give the button a link.